docs/userguide/networking/get-started-overlay.md

```diff
@@ -11,6 +11,36 @@ weight=-3
 
 # Get started with multi-host networking
 
+As of Docker Engine 1.12 and later, you can create an overlay network on a manager node running in [swarm mode](../../swarm/swarm-mode.md).
+
+The swarm makes the overlay network available only to nodes in the swarm that
+require it for a service. When you create a service that uses the overlay
+network, the manager node automatically extends the overlay network to nodes
+that run service tasks.
+
+To learn more about running Docker Engine in swarm mode, refer to the
+[Swarm mode overview] (../../swarm/index.md).
+
+The example below shows how to create a network and use it for a service from a manager node in the swarm:
+
+```bash
+# Create an overlay network `my-multi-host-network`.
+$ docker network create --driver overlay my-multi-host-network
+
```

Comment:
@SvenDowideit we've been adding blank lines between the command and output, because we received feedback that it's not always clear where the command ends, and the output begins. Adding the blank line was received as being more clear, so we're trying to use that now.

I think we can try this, and evaluate if it was a good decision (otherwise we just remove them everywhere)